source/_integrations/aws_s3.markdown

```
@@ -28,6 +28,80 @@
+            "s3:PutObject",
+            "s3:DeleteObject"
+          ],
+          "Resource": [
+            "arn:aws:s3:::YOUR_BUCKET_NAME",
+            "arn:aws:s3:::YOUR_BUCKET_NAME/*"
+          ]
+        }
+      ]
+    }
+    ```
+4. Click **Next**, give the policy a name like `HomeAssistantS3Policy`, and create it.
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 3; Actual: 4; Style: 1/2/3

(MD029, ol-prefix)